.travis.yml

```diff
@@ -5,8 +5,10 @@ compiler:
 addons:
  apt:
   packages:
+   - pkg-config
    - libgraphite2-dev
    - libicu-dev
+   - libicu52
```

Comment:
@srl295 Besides what @khaledhosny said, this was a test to see if I could get Travis to find ICU at all. So far no go on the containerized system, although following @behdad's lead from harfbuzz, the non-containerized Travis images for Ubunty Trusty find ICU out of the box.